renderer/views/player.js

```diff
@@ -43,7 +43,7 @@ function renderMedia (state) {
       mediaElement.play()
     }
     // When the user clicks or drags on the progress bar, jump to that position
-    if (state.playing.jumpToTime) {
+    if (state.playing.jumpToTime != null) {
```

Comment:
I guess the single equals is a fancy way to check for null or undefined. Standard allows it so 👍